prawcore/sessions.py

```diff
@@ -38,6 +40,7 @@ class Session(object):
                          codes['unauthorized']: authorization_error_class,
                          codes['unavailable_for_legal_reasons']:
                          UnavailableForLegalReasons,
+                         codes['read_timeout']: ReadTimeout,
```

Comment:
Can you keep this in sorted order? Also it looks like the `ReadTimeout` class doesn't exist in PRAW.

Reply:
Oh right, I've added the ReadTimeout class to the PRAW exceptions (inherits from RequestException).